landr.conf.js

```diff
@@ -0,0 +1,62 @@
+const prepAssets = require('./www/prepAssets')
+
+module.exports = {
+  theme: 'landr-theme-basic',
```

Comment:
Where is the source object that contains the extracted data from the repo that this theme is using?

Reply:
I've written it to www/locals.json for now.